### ISSUE TITLE
Using Anko without Android

### ISSUE THREAD

**Issue by yar00001** [Kotlin/anko#644]
Is there any possible implementation of Anko which could be used without Android libraries? I am seeing Android Context is used in `ManagedSQLiteOpenHelper`.

**Comment by hendrawd** [comment]
I don't think you should use anko in the first place, because like the description: "Pleasant Android application development", anko purpose is for supporting android development
